tests/misc_env/lvm_deployer.py

```diff
@@ -0,0 +1,86 @@
+from utility.log import Log
```

Comment:
could we rename the module to `create_lvms`? I believe that it is more appropriate

Reply:
we have `lvm_utils.py` already, create_lvms might confuse us. But we can think of another name.